test/e2e/api/info.test.ts

```diff
@@ -0,0 +1,38 @@
+import { Application } from 'express';
```

Comment:
I think it's unused. Please fix your tslint or ts => https://palantir.github.io/tslint/rules/no-unused-variable/